src/Ng/Jobs/LightningJob.cs

```diff
@@ -146,20 +172,14 @@ protected override void Init(IDictionary<string, string> arguments, Cancellation
                     InitStrike(arguments);
                     break;
                 default:
-                    throw new ArgumentNullException();
+                    throw new NotSupportedException($"The lightning command '{_command}' is not supported.");
             }
-
-            _contentBaseAddress = arguments.GetOrThrow<string>(Arguments.ContentBaseAddress);
-            _storageAccount = arguments.GetOrThrow<string>(Arguments.StorageAccount);
-            _storageContainer = arguments.GetOrThrow<string>(Arguments.StorageContainer);
-            _storageBaseAddress = arguments.GetOrThrow<string>(Arguments.StorageBaseAddress);
-            _compress = arguments.GetOrDefault(Arguments.Compress, false);
         }
 
         private void InitPrepare(IDictionary<string, string> arguments)
         {
             _outputFolder = arguments.GetOrThrow<string>(Arguments.OutputFolder);
-            _catalogIndex = arguments.GetOrThrow<string>(Arguments.CatalogIndex);
+            _catalogIndex = arguments.GetOrThrow<string>(Arguments.Source);
```

Comment:
IMO all the jobs should use `CatalogIndex` instead of `Source`, because it's confusing what `Source` refers to unless you're already familiar with the jobs, but I like aligning this with the other jobs.